payment/views.py

```diff
@@ -198,10 +199,12 @@ def accept_payment_terms(request):
     )
     terms.accept(request.user)
     data['accepted'] = terms.version
-
+    if 'next' in request.GET:
+        redirect_to = request.GET['next']
+    else:
+        redirect_to = reverse('dashboard')
```

Comment:
As for forgiveness is better than ask for permission?
```suggestion
    try:
        redirect_to = request.GET['next']
    except KeyError:
        redirect_to = reverse('dashboard')
```